### PR TITLE
Update rxjs refs in doc

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Links to RxJS doc out of date in Observables guide,
reference needed for RxJS `Subject` in API doc for `EventEmitter`

Issue Number: https://github.com/angular/angular/issues/29499

## What is the new behavior?
Link to current doc added in API doc for EventEmitter and in Observables guide pages.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/rx-library.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The RxJS library
 
-Reactive programming is an asynchronous programming paradigm concerned with data streams and the propagation of change ([Wikipedia](https://en.wikipedia.org/wiki/Reactive_programming)). RxJS (Reactive Extensions for JavaScript) is a library for reactive programming using observables that makes it easier to compose asynchronous or callback-based code ([RxJS Docs](http://reactivex.io/rxjs/)).
+Reactive programming is an asynchronous programming paradigm concerned with data streams and the propagation of change ([Wikipedia](https://en.wikipedia.org/wiki/Reactive_programming)). RxJS (Reactive Extensions for JavaScript) is a library for reactive programming using observables that makes it easier to compose asynchronous or callback-based code. See ([RxJS Docs](https://rxjs-dev.firebaseapp.com/guide/overview)).
```
Comment (StephenFluin) [inline aio/content/guide/rx-library.md]: Shouldn't this be rxjs.dev?

File: packages/core/src/event_emitter.ts [hunk 2]
```diff
@@ -11,12 +11,16 @@
...
- * or asynchronously, and register handlers for those events by subscribing
- * to an instance.
+ * Use in components with the `@Output` directive to emit custom events
+ * synchronously or asynchronously, and register handlers for those events
+ * by subscribing to an instance.
  *
  * @usageNotes
  *
+ * Extends
+ * [RxJS `Subject`](https://rxjs-dev.firebaseapp.com/api/index/class/Subject)
```
Comment (StephenFluin) [inline packages/core/src/event_emitter.ts]: Shouldn't this be rxjs.dev?

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: aio/content/guide/rx-library.md (hunk 1 above)
Comment (jbogarthyde) [reply aio/content/guide/rx-library.md]: No, this is the correct link.

**Comment by StephenFluin** [comment]
@jbogarthyde I'm pretty sure (I checked with a few folks) that anywhere you are using rxjs-dev.firebaseapp.com, we should be using https://rxjs.dev/ instead

File: aio/content/guide/rx-library.md (hunk 1 above)
Comment (jbogarthyde) [reply aio/content/guide/rx-library.md]: Right - they go to the same page. Changing all occurences.

**Comment by jbogarthyde** [comment]
Right -- the new link goes to the same pages - corrected it, @StephenFluin  please check.

**Review by StephenFluin — APPROVED** [review]

**Comment by jbogarthyde** [comment]
Just needs code-owner approval

File: aio/content/guide/observables-in-angular.md [hunk 3]
```diff
@@ -2,13 +2,15 @@
 
 Angular makes use of observables as an interface to handle a variety of common asynchronous operations. For example:
 
-* The `EventEmitter` class extends `Observable`.
+* The `EventEmitter` class lets you use observables with the Angular `@Output` decorator.
```
Comment (trotyl) [inline aio/content/guide/observables-in-angular.md]: This sounds confusing, any observable can be used for Angular `@Output` decorator without knowing `EventEmitter`, and `EventEmitter` is only meant for `Observer` role of `Subject`, while the `Observable` role is hidden from user, there should never be a "let user use observables" in `EventEmitter`.

**Review by StephenFluin — APPROVED** [review]

File: aio/content/guide/rx-library.md (hunk 1 above)
Comment (IgorMinar) [reply aio/content/guide/rx-library.md]: no, actually, this should be https://rxjs.dev/guide/overview. @jbogarthyde can you please update it?

**Pull request closed without merging**